src/View/Helper/TimeHelper.php

```diff
@@ -326,8 +326,12 @@ public function format($date, $format = null, $invalid = false, $timezone = null
      * @throws \InvalidArgumentException When the date cannot be parsed
      * @see \Cake\I18n\Time::i18nFormat()
      */
-    public function i18nFormat($date, $format = null, $invalid = false, $timezone = null)
+    public function i18nFormat($date = null, $format = null, $invalid = false, $timezone = null)
```

Comment:
Good point. The intention was to return the ```$invalid``` argument in the event nothing or null is passed into the ```$date```  argument. However, that doesn't seem to make sense since ```$date``` is a required argument.